doc/rados/operations/health-checks.rst

```diff
@@ -141,6 +141,14 @@ To adjust the warning threshold, run the following command:
 
    ceph config set global mon_data_size_warn <size>
 
+MON_NETSPLIT
+____________
+
+A network partition has occurred between the monitors. This health check is
+raised when one or more monitors detect that at least one pair of monitors
```

Comment:
This connotes that mons are explicitly paired in some fashion.  Perhaps _that at least two Ceph Monitors have lost connectivity or reachability,_